metaworld/envs/mujoco/sawyer_xyz/sawyer_xyz_env.py

```diff
@@ -305,3 +435,78 @@ def _get_state_rand_vec(self):
                 size=self._random_reset_space.low.size)
             self._last_rand_vec = rand_vec
             return rand_vec
+
+    def _gripper_caging_reward(self, action, obj_position, obj_radius):
```

Comment:
If this gripper caging reward is different from what's been tested/benchmarked already, I think it's probably best to put it in just the sweep environment for now. If we find that it's useful for other environments as we build out the reward functions, then we can always copy it to sawyer_xyz_env later. It just may be premature now. @avnishn thoughts?

Reply:
I think we should stick it inside the environments that it is used in for now, and then rename it and move it to reward utils later @adibellathur @haydenshively